### DISCUSSION TITLE
Guidance on refreshing a trained model with new data

### DISCUSSION THREAD

**Discussion by stdp95** [pymc-labs/pymc-marketing#1005] [General]
Hi,

Thank you for developing and maintaining pymc-marketing!

I have a model that I’ve already trained using pymc-marketing, but I now have a new set of data that I’d like to use to refresh the model. I’m wondering what the best approach is for incorporating this new data. Should I retrain the entire model from scratch with the combined old and new data, or is there an incremental update approach that I can take to avoid rebuilding the model entirely?

Any guidance on efficiently updating the model with new data without starting over would be greatly appreciated.

Thank you so much for your attention and participation.

**Comment by williambdean** [comment]
Rebuild with all of the data is recommended approach
